chai/src/main/kotlin/com/droidconke/chai/assets/AssetProvider.kt

```diff
@@ -0,0 +1,19 @@
+package com.droidconke.chai.assets
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.drawable.BitmapDrawable
+import androidx.core.graphics.drawable.toBitmap
+import com.droidconke.chai.exceptions.attemptAssetRetrieval
+import java.io.InputStream
+
+object AssetProvider {
+
+    fun image(ctx: Context, path: String): Bitmap =

```

Comment:
I want to add support for other asset types, using generics to reduce the amount of redundancy in creating a bunch of new functions